src/mctp-netlink.c

```diff
@@ -709,21 +711,33 @@ static int parse_getlink_dump(mctp_nl *nl, struct nlmsghdr *nlh, uint32_t len)
 			/* Skip non-MCTP interfaces */
 			continue;
 		}
-		if (!mctp_get_rtnlmsg_attr_u32(IFLA_MCTP_NET, rt_mctp, mlen, &net)) {
-			warnx("Missing IFLA_MCTP_NET");
-			continue;
-		}
-
-		/* TODO: media type */
 
 		ifname = mctp_get_rtnlmsg_attr(IFLA_IFNAME, rta, rlen, &ifname_len);
 		if (!ifname) {
 			warnx("no ifname?");
 			continue;
 		}
+
 		ifname_len = strnlen(ifname, ifname_len);
+		if (!mctp_get_rtnlmsg_attr_u32(IFLA_MCTP_NET, rt_mctp, mlen, &net)) {
+			warnx("Missing IFLA_MCTP_NET for %s", ifname);
+			continue;
+		}
+
+		if (!mctp_get_rtnlmsg_attr_u32(IFLA_MIN_MTU, rta, rlen, &min_mtu)) {
+			warnx("Missing IFLA_MIN_MTU for %s", ifname);
+			continue;
+		}
+
+		if (!mctp_get_rtnlmsg_attr_u32(IFLA_MAX_MTU, rta, rlen, &max_mtu)) {
+			warnx("Missing IFLA_MAX_MTU for %s", ifname);
+			continue;
+		}
```

Comment:
`lo` reports:

```
[{nla_len=8, nla_type=IFLA_MTU}, 65536], [{nla_len=8, nla_type=IFLA_MIN_MTU}, 0], [{nla_len=8, nla_type=IFLA_MAX_MTU}, 0]
```

Do we need any special handling for zero?

Reply:
A max MTU of 0 will result in the link MTU being set to 0, the same as currently, so that should be fine. (If properties were optional we could omit min_mtu/max_mtu if they were 0, but it doesn't seem worth adding another interface just for that special case)